docs/advanced-customization/internals.md

```diff
@@ -14,15 +14,15 @@ This component follows [JSON Schema](http://json-schema.org/documentation.html)
 
     The `anyOf` and `oneOf` keywords are supported; however, properties declared inside the `anyOf/oneOf` should not overlap with properties "outside" of the `anyOf/oneOf`.
 
-    You can also use `oneOf` with [schema dependencies](../usage/dependencies.md) to dynamically add schema properties based on input data.
+    You can also use `oneOf` with [schema dependencies]([../usage/dependencies.md](https://react-jsonschema-form.readthedocs.io/en/docs/usage/dependencies/#schema-dependencies)) to dynamically add schema properties based on input data.
```

Comment:
I'm not sure these links work... I clicked on the `view file` to get to the rendered `.md` file and clicked the link and got a 404

Reply:
@heath-freenome my bad... I just fixed the links, can you check again?